libs/dh/wholesale/domain/src/streamToFile.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Observable } from 'rxjs';
+
+interface Options {
+  /** The name of the file. */
+  name: string;
+  /** The MIME type of the file. */
+  type: string;
+}
+
+/** Streams data to a file for downloading. */
+export const streamToFile =
+  ({ name, type }: Options) =>
+  (data: unknown): Observable<void> =>
+    new Observable((observer) => {
+      const blobPart = data as BlobPart;
+      const blob = new Blob([blobPart], { type });
+      const basisData = window.URL.createObjectURL(blob);
```

Comment:
Question: is URL, not top level?

Reply:
Good point, I just copied the code from the store. I have removed the `window` prefix 👍